src/System.ComponentModel.TypeConverter/src/System/ComponentModel/BindableAttribute.cs

```diff
@@ -145,7 +145,7 @@ public override int GetHashCode()
         /// <internalonly/>
         public override bool IsDefaultAttribute()
         {
-            return (this.Equals(Default) || _isDefault);
+            return (Equals(Default) || _isDefault);
```

Comment:
These parentheses are redundant (there are other instances, but I'll just leave the one comment)